src/test/java/org/terasology/launcher/game/TestGameStarter.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.launcher.game;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.terasology.launcher.util.JavaHeapSize;
+import org.terasology.launcher.util.LogLevel;
+
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.terasology.launcher.Matchers.hasItemsFrom;
+
+public class TestGameStarter {
+    static final String JAVA_ARG_1 = "-client";
+    static final String JAVA_ARG_2 = "--enable-preview";
+    static final String GAME_ARG_1 = "--no-splash";
+    static final String GAME_DIR = "game";
+    static final String GAME_DATA_DIR = "game_data";
+    static final JavaHeapSize HEAP_MIN = JavaHeapSize.NOT_USED;
+    static final JavaHeapSize HEAP_MAX = JavaHeapSize.GB_4;
+    static final LogLevel LOG_LEVEL = LogLevel.INFO;
```

Comment:
We just removed `LogLevel` so this has to be adjusted to use slf4j's `Level` - should be an easy fix.